piyushgarg-dev/review-app
# feat: Designed and added landing page

## What does this PR do?


This PR is to successfully add the Landing page along with animations and a visually appealing UI. Made necessary changes to improve the SEO of the page by adding meta tags which describe briefly what the application does and same with alt attributes 
of Image tags to help the SE understand the picture.
Fixes #7


Loom Video : https://www.loom.com/share/6270e54ee46749edb80755eabdb849d1?sid=4499a743-53c5-429f-9fe3-b72694a7d497

## Requirement/Documentation





## Type of change






## How should this be tested?



- No unique test cases required.
- Reload the browser to see the initial animations.


## Mandatory Tasks
